recipes/libsvtav1/all/conanfile.py

```diff
@@ -0,0 +1,96 @@
+import os
+from conan import ConanFile
+from conan.tools.cmake import cmake_layout, CMakeToolchain, CMakeDeps, CMake
+from conan.tools.files import copy, get, rmdir
+
+required_conan_version = ">=1.47.0"
+
+
+class SVTAV1Conan(ConanFile):
+    name = "libsvtav1"
+    license = "BSD-3-Clause"
+    description = "An AV1-compliant software encoder/decoder library"
+    topics = "av1", "codec", "encoder", "ffmpeg", "video"
+    homepage = "https://gitlab.com/AOMediaCodec/SVT-AV1"
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "build_encoder": [True, False],
+        "build_decoder": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "build_encoder": True,
+        "build_decoder": True,
+    }
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
```

Comment:
If you're okay bumping the minimum required version to 1.53, you can use `rm_safe` in the configure method.

```suggestion
            self.options.rm_safe("fPIC")
```